spec/validators_spec.rb

```diff
@@ -0,0 +1,66 @@
+require_relative '../lib/validators.rb'
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+
+describe Validators do
+  curr_board = Board.new
+  player = Player.new('sam', 'X')
+  move = 3
```

Comment:
[Optional]

I think we can use `let` helper method to wrap the variables here, this ensures that the one test case does not interfere with another.



> Students should [Use `let` instead](https://github.com/rubocop-hq/rspec-style-guide#let-blocks). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/before_each.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))